console/console-init/ui/src/components/ProjectHeaderCard/ProjectHeaderCard.tsx

```diff
@@ -43,127 +41,125 @@ const ProjectHeaderCard: React.FunctionComponent<IProjectHeaderCardProps> = ({
   msgCount
 }) => {
   return (
-    <PageSection>
-      <Gallery gutter="sm">
+    <Gallery gutter="sm">
```

Comment:
Instead of writing everything in the same component, divide this component as per the concern and import the smaller components here.